doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -254,6 +254,8 @@ Serial LTE modem
 nRF5340 Audio
 -------------
 
+* Added support for Filter Accept List.
```

Comment:
Suggest we add some more info. E.g. that this is enabled by default and how it influences the user experience.